public/app/core/components/AppChrome/AppChromeService.tsx

```diff
@@ -28,9 +29,10 @@ export class AppChromeService {
 
   readonly state = new BehaviorSubject<AppChromeState>({
     chromeless: true, // start out hidden to not flash it on pages without chrome
-    sectionNav: { text: t('nav.home.title', 'Home') },
+    sectionNav: { node: { text: t('nav.home.title', 'Home') }, main: { text: '' } },
     searchBarHidden: store.getBool(this.searchBarStorageKey, false),
     kioskMode: null,
+    layout: PageLayoutType.Standard,
```

Comment:
little unsure about making the default `Standard`. suspect it would be safer to make this `Custom`? worried about children that aren't pages... 🤔

Reply:
yea, could maybe cause some flickering section nav?

Reply:
I tested this on login page and canvas seems to work without any flickering,